k/Kivy.py

```diff
@@ -0,0 +1,14 @@
+import kivy
+kivy.require('1.0.6')
+
+from kivy.app import App
+from kivy.uix.label import Label
+
```

Comment:
There should only be one newline space instead of two

Reply:
Done.